Update Vue syntax files

sebthom opened the pull request.

vrubezhny commented:
An IP Team Review request is created:

- [ ] [-/-/vuejs/language-features-subset/v2.2.8](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/21575)

sebthom commented:
> @sebthom the folder `language-configurations/vue/` in the PR doesn't add any new language configurations while the existing ones [in master] don't seem to be the same as in [vuejs/language-tools@`v2.2.8`/extensions/vscode/languages](https://github.com/vuejs/language-tools/tree/v2.2.8/extensions/vscode/languages?rgh-link-date=2025-06-05T15%3A10%3A25Z) .
> 
> Could you please check if these configurations need to be added to the PR? 

@vrubezhny I don't understand what you mean. I just verified it again, all three locations contain the same language configuration files last modified two years ago.
- https://github.com/eclipse-wildwebdeveloper/wildwebdeveloper/tree/master/org.eclipse.wildwebdeveloper/language-configurations/vue
- https://github.com/vuejs/language-tools/tree/master/extensions/vscode/languages
- https://github.com/vuejs/language-tools/tree/v2.2.8/extensions/vscode/languages

The only thing I added as part of this PR is the missing license file and I updated the readme to point to a tag instead of master so that we know which version actually was copied.

vrubezhny reviewed (approved):
Looks good to me, thanks!
